client/components/LatestBid.jsx

```diff
@@ -0,0 +1,18 @@
+/*
+    ./client/components/LatestBid.jsx
+*/
+import React from 'react';
+
+export default class LatestBid extends React.Component {
+
+	render(props) {
+
+		return (
+			<div>
+				<span>Rs. {this.props.highestBid}</span>
```

Comment:
You can eliminate `<div>` wrapping and just return `<span>` element. [Semantic HTML]